features/fixtures/maze_runner/Assets/Scripts/Main.cs

```diff
@@ -36,6 +36,19 @@ public class Main : MonoBehaviour
 
 #endif
 
+#if UNITY_SWITCH
+
+    [DllImport("__Internal")]
+    private static extern int bugsnag_getArgsCount();
+
+    [DllImport("__Internal")]
+    private static extern string bugsnag_getArg(int index);
+
+    private SwitchCacheType _switchCacheType = SwitchCacheType.R;
+    private int _switchCacheIndex = 0;
+    private string _switchCacheMountName = string.Empty;
```

Comment:
```suggestion
    private string _switchCacheMountName = "BugsnagCache";
```
This is probably why that mount name error happened

Reply:
Ah, so a value is needed and the "else" value I put on L369 of unity_steps.rb would not be needed if we change this.

Reply:
I've changed both